Gemfile

```diff
@@ -8,6 +8,8 @@ group(:release, optional: true) do
   gem 'octokit', '~> 4.18.0'
 end
 
+gem 'packaging', require: false
```

Comment:
We can also add the gem to the `:release` group, but then we'd have to update the CJC artifact generation step to run `bundle install --with release`.

Reply:
Don't really have a strong opinion from this, adding it to the release group would enable us to install it only when needed(with the change to cjc that you mentioned, since that is the only place where it is used.)